server/lib/s3backup/__init__.py

```diff
@@ -0,0 +1,176 @@
+"""
+Simple module level convenience functions.
```

Comment:
More descriptive comments please: something like "This module provides convenience functions that interface to lower-level services, provided by the boto3 module."